verl/trainer/config/ppo_trainer.yaml

```diff
@@ -129,7 +129,7 @@ actor_rollout_ref:
       n: 1
       do_sample: False # default eager for validation
     multi_turn:
-      enable: False  # should set rollout.name to sglang_async if True
+      enable: False  # should set rollout.name to sglang if True
```

Comment:
comment could improved: should set when handling multi-turn tool interaction tasks.